api/placer.py

```diff
@@ -475,3 +477,32 @@ def finalize(self):
             'event': 'result',
             'data': result,
         })
+
+class AnalysisPlacer(Placer):
+
+    def check(self):
+        self.requireMetadata()
+        #validators.validate_data(self.metadata, 'analysys.json', 'input', 'POST', optional=True)
+        self.saved = []
+
+    def process_file_field(self, field, info):
+        self.save_file(field)
```

Comment:
If `info` is now an optional parameter to `save_file`, may as well pass it along.